### ISSUE TITLE
Question: Tenant signing certificates

### ISSUE THREAD

**Issue by a-tok** [IdentityServer/IdentityServer3#2452]
Hello,

Is it possible for each tenant to have a different signing certificate? 
If so, how would one accomplish this? 

Any help is greatly appreciated.

Thank You

**Comment by brockallen** [comment]
What's the definition of a "tenant"?

**Comment by a-tok** [comment]
The value  provided when defining acr_values e.g. ...&acr_values=tenant:abc.

**Comment by brockallen** [comment]
If you want different signing certs, then you need different instances of the IdSvr middleware. That would only really work if you had different URLs for the different instances.

So no, we don't really support anything like what you're asking.

**Comment by a-tok** [comment]
I see, this would be a cool feature for the sake of flexibility and giving devs options. 

I just started learning OWIN, does IAppBuilder.Map() create new middleware?  I was able to get different local UserServices running on a single host by using IAppBuilder.Map() with separate instances of idsrv options per path. Someone suggested that I should use acr_values and tenants instead, so you kind of do support multiple signing certificates, just not for tenants.
